release-notes/opensearch-documentation-release-notes-2.9.0.md

```diff
@@ -0,0 +1,30 @@
+# OpenSearch Documentation Website 2.9.0 Release Notes
+
+The OpenSearch 2.9.0 documentation includes the following additions and updates.
+
+## New documentation for 2.9.0
+
+- Change Neural Search plugin from experimental to GA. [#4588](https://github.com/opensearch-project/documentation-website/pull/4588)
+- [DOC] Add new property setting fast_double_writer to install documentation  [#4580](https://github.com/opensearch-project/documentation-website/pull/4580)
+- Add Spark connector settings [#4577](https://github.com/opensearch-project/documentation-website/pull/4577)
+- Add Connectors and ML updates for 2.9 [#4554](https://github.com/opensearch-project/documentation-website/pull/4554)
+- Search pipeline GA documentation [#4553](https://github.com/opensearch-project/documentation-website/pull/4553)
+- Add documentation for OCSF field mapping and correlation engine API [#4549](https://github.com/opensearch-project/documentation-website/pull/4549)
+- Add search pipeline metrics to Nodes Stats API [#4548](https://github.com/opensearch-project/documentation-website/pull/4548)
+- [DOC] Add Integrations new documentation [#4546](https://github.com/opensearch-project/documentation-website/pull/4546)
+- Add thread level details to _tasks API [#4542](https://github.com/opensearch-project/documentation-website/pull/4542)
```

Comment:
```suggestion
- Add thread level details to the `_tasks` API [#4542](https://github.com/opensearch-project/documentation-website/pull/4542)
```

Reply:
The automatic workflow simply takes PR names. Since tic marks are not rendered in PR headers, they are not included in the PR name and thus not propagated here.